Data-Structures/Array/test/firstDuplicate.test.js

```diff
@@ -0,0 +1,46 @@
+import firstDuplicate from '../firstDuplicate'
+
+describe('firstDuplicate test', function () {
```

Comment:
Consider using arrow functions for syntactical consistency with most (?) other tests.